Backend/src/model/course.py

```diff
@@ -1,13 +1,93 @@
+from __future__ import annotations
 from dataclasses import dataclass
-from typing import List
+from typing import Callable, List
 from .section import Section
+from .date import Date
+from .day_of_week import DayOfWeek
+from .filter import Filter
 
 @dataclass
 class Course:
+    code: str
     name: str
-    title: str
```

Comment:
Just keep in mind the current db schema uses "Title"

Reply:
I will revert this in a new pr